### PR TITLE
Fixup READMEs and minor ansible tweaks

### PR DESCRIPTION
**Pull request by StephenBrown2**
Thanks for translating this to vagrant! I wasn't able to get it running on my mac, but since I'm a bit of a stickler for proper formatting, here's a PR to that regard.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,136 +1,135 @@
...
 
-```shell script
+On MacOS, Linux and Windows:
+
+```sh
 ./build.sh
 ```
 
-This will, by default, build and provision an OpenSUSE Leap 15.2 vagrant box in libvirt - including rsyncing the source to the 
-VM in /root. It will then build and install Rockstor from source as per the install guide in the top level README.md.
```
Comment (StephenBrown2) [inline README.md:62]: I couldn't locate this README, so I removed this phrase, but if we know where it is, we should link to it.

Comment (phillxnet) [reply README.md:62]: Agreed.

**Comment by phillxnet** [comment]
@StephenBrown2 Thanks for the fix-ups here. Much appreciated. I'd like to take a better look later so will hopefully get this merged then.

Again, thanks for stepping up to help get this new repo off the ground.

**Comment by phillxnet** [comment]
@StephenBrown2 This all looks nice. Thanks.

Hopefully this repo can attract enough attention to take over from our current by-hand wall-of-text approach in the referenced wiki forum thread:
https://forum.rockstor.com/t/built-on-opensuse-dev-notes-and-status/5662
Thanks again for your initial offering on the forum and for updating/tweaking @mikemonkers initial/opening pull request to this repo.

@FroggyFlox I'm merging this as-is given this repo is in a fledgling and non working/tested state and this can only help.

**Pull request merged**